apiserver/common/networkingcommon/networkconfigapi.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// The networkconfigapi package implements the network config parts
+// common to machiner and provisioner interface
+
+package networkingcommon
+
+import (
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/errors"
```

Comment:
github.com/juju/errors is considered an 'external' package because it isn't under github.com/juju/juju